clio-server/src/main/resources/reference.conf

```diff
@@ -8,14 +8,16 @@
 include "clio-server-version"
 
 clio.server {
-  environment: dev
-
   http-server {
     interface: 0.0.0.0
     port: 8080
     shutdown-timeout: 60s
   }
 
+  persistence {
```

Comment:
Why not `clio.persistence`?

Could this attribute be placed directly under `clio` instead, e.g. `clio.service-account-json`? Or does it need to be defined as a top-level item?

Reply:
I think `clio.server.persistence` makes more sense than `clio.persistence` since all the GCS-writing is going to happen within the server. I made the `persistence` wrapper around the account-JSON setting in case we end up needing to add more configuration around how we write to / read from the buckets.

Reply:
OK. I know we will have different bucket locations based on environment, so perhaps that will go here.

Although in this case we aren't using the service account for persistence, but for authentication, correct?

Reply:
Authentication-for-persistence, so yes and no :)

An alternate setup I looked at was how Cromwell does things, where they have a separate [config block](https://github.com/broadinstitute/cromwell#configuring-authentication) for accounts in general, and then other spots in config can refer to accounts by-name. I decided against it because it seemed like overkill for what we're trying to do.